worlds/dark_souls_3/__init__.py

```diff
@@ -86,6 +86,10 @@ def generate_early(self):
             self.enabled_location_categories.add(DS3LocationCategory.NPC)
         if self.multiworld.enable_key_locations[self.player] == Toggle.option_true:
             self.enabled_location_categories.add(DS3LocationCategory.KEY)
+            if self.multiworld.early_banner[self.player] == EarlySmallLothricBanner.option_early_global:
+                self.multiworld.early_items[self.player]['Small Lothric Banner'] = 1
+            if self.multiworld.early_banner[self.player] == EarlySmallLothricBanner.option_early_local:
```

Comment:
This should probably be an `elif`.

```suggestion
            elif self.multiworld.early_banner[self.player] == EarlySmallLothricBanner.option_early_local:
```